lib/project_types/script/config/extension_points.yml

```diff
@@ -3,13 +3,16 @@ discount:
     package: "@shopify/extension-point-as-discount"
     version: "~0.0.21"
     sdk-version: "^4.0.0"
+    toolchain-version: "^0.0.1"
 unit_limit_per_order:
   assemblyscript:
     package: "@shopify/extension-point-as-unit-limit-per-order"
     version: "~0.0.21"
     sdk-version: "^4.0.0"
+    toolchain-version: "^0.0.1"
 payment_filter:
   assemblyscript:
     package: "@shopify/extension-point-as-payment-filter"
     version: "~0.0.5"
-    sdk-version: "^4.0.0"
\ No newline at end of file
+    sdk-version: "^4.0.0"
+    toolchain-version: "^0.0.1"
```

Comment:
Should probably use `~` instead of `^` unless we want to change the toolchain version to something like `0.1.0` so we get new patch versions automatically. Actually that may be a good idea anyway as we try and navigate the 0.9.x to 0.10.x upgrade for AssemblyScript.